src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Amazon Lambda Telemetry integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's Amazon App Runner monitoring integration: what data it reports, and how to enable it."
+---
+
+ # New Relic AWS Lambda Telemetry extension
+
+The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch.Send your logs, events, metrics and traces by simply adding the extension as the layer and inputting your New Relic license key.
+
+## Installation
+
+1. In the AWS Console go to **Lambda** > **Functions** then choose the function you'd like to ingest data from.
+2. Select **Layers** > **Add Layer**.
+3. In **Choose a layer** select the **Specify a layer by providing the ARN** option.
+4. Put `arn:aws:lambda:us-west-2:533243300146:layer:python-telemetry-api-extension:<version>` in the field, then click **Add**.
```

Comment:
the name of the function has changed, now it is  `arn:aws:lambda:us-west-2:533243300146:layer:AwsLambdaExtension:<version>`